### ISSUE TITLE
Aria attributes are missing from the widget

### ISSUE THREAD

**Issue by stephanbruijnis** [stephanbruijnis/mx-multiSelectEnum#2]
Add accessibility to the widget
https://docs.mendix.com/howto/extensibility/create-a-pluggable-widget-two/#35-adding-accessibility
